README.md

```diff
@@ -2,8 +2,52 @@
 
 ## Purpose
 
-_... help fill this in!_
+Purpose is *not* to respond to reports of security issues with Node.js, that is
+done by the "security team", see https://github.com/nodejs/node#security
+
+*todo*: should the team be identified in
+https://github.com/nodejs/node#security?
```

Comment:
I mean the "security team", not the security working group. I might raise this as a bug on nodejs. Right now the security team is secret, I'm not sure that was intended.